sdk/spring/azure-spring-boot/pom.xml

```diff
@@ -166,7 +166,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.0.9</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <version>1.1.0-beta.7</version> <!-- {x-version-update;beta_com.azure:azure-identity;dependency} -->
```

Comment:
why are we doing this only for spring boot ?

Reply:
Reverted spring boot, not needed here.